<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fiche_absence</class>
 <widget class="QDialog" name="fiche_absence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>651</width>
     <height>421</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>dgd</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>ajout_absence</string>
    </attribute>
    <widget class="QRadioButton" name="justifieRB">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>absence justifiée</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="nonjustifieRB">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>absence nom justifiée</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CIN_EmployeeLE">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="CIN_EmployeeLB">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Cin Employee&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="causeLB">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Cause&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="causeCB">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>141</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Maladie</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Maternite</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="ajouterabs">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>290</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
     <property name="icon">
      <iconset resource="Images.qrc">
       <normaloff>:/icon/Icon/112.png</normaloff>:/icon/Icon/112.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>651</width>
       <height>491</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Images.qrc">:/Arriere/images/fond1.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Images.qrc">
       <normaloff>:/icon/Icon/28.png</normaloff>:/icon/Icon/28.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>justifieRB</zorder>
    <zorder>nonjustifieRB</zorder>
    <zorder>CIN_EmployeeLE</zorder>
    <zorder>CIN_EmployeeLB</zorder>
    <zorder>causeLB</zorder>
    <zorder>causeCB</zorder>
    <zorder>ajouterabs</zorder>
    <zorder>pushButton</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>afficher_absence</string>
    </attribute>
    <widget class="QLabel" name="cin_employeeLB">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Cin Employee&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cin_employeeLE">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>531</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="afficherabs">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>340</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>afficher</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>651</width>
       <height>491</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Images.qrc">:/Arriere/images/fond1.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Images.qrc">
       <normaloff>:/icon/Icon/28.png</normaloff>:/icon/Icon/28.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>cin_employeeLB</zorder>
    <zorder>cin_employeeLE</zorder>
    <zorder>tableView</zorder>
    <zorder>afficherabs</zorder>
    <zorder>pushButton_2</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>supprimer_absence</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Cin Employee&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cin_employeeLE_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimerabs">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
     <property name="icon">
      <iconset resource="Images.qrc">
       <normaloff>:/icon/Icon/118.png</normaloff>:/icon/Icon/118.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="retoursup">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Images.qrc">
       <normaloff>:/icon/Icon/28.png</normaloff>:/icon/Icon/28.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>651</width>
       <height>471</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Images.qrc">:/Arriere/images/fond1.png</pixmap>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>label</zorder>
    <zorder>cin_employeeLE_2</zorder>
    <zorder>supprimerabs</zorder>
    <zorder>retoursup</zorder>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>rechercher</string>
    </attribute>
    <widget class="QPushButton" name="OKpB">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Images.qrc">
       <normaloff>:/icon/Icon/156.png</normaloff>:/icon/Icon/156.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableView" name="RechView">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>101</y>
       <width>431</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="AnnulerpB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Images.qrc">
       <normaloff>:/icon/Icon/28.png</normaloff>:/icon/Icon/28.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Refla">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>cin employee</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cin_employeeLE_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>651</width>
       <height>451</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Images.qrc">:/Arriere/images/fond1.png</pixmap>
     </property>
    </widget>
    <zorder>label_5</zorder>
    <zorder>OKpB</zorder>
    <zorder>RechView</zorder>
    <zorder>AnnulerpB</zorder>
    <zorder>Refla</zorder>
    <zorder>cin_employeeLE_3</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
